README.md

```diff
@@ -22,7 +22,11 @@ Please notice that the extension uses `.vscode/.react` directory at the project
 
 ## Debugging React Native applications
 
-Click the debug icon ![Choose React Native debugger](images/debug-view-icon.png) in the View bar, and then click the configuration (gear) icon ![Configure-gear](images/configure-gear-icon.png), then choose the React Native debug environment.
+To start debugging you need to create debug configurations in `.vscode/launch.json` first. You can always do it by opening it and clicking at `Add Configuration...` button and then choosing needed debug configuration.
+
+![Choose React Native debug configuration](images/add-debug-configuration.png)
+
+In case if you haven't `.vscode/launch.json` file yet, you can add a whole default debug configuration set. To do that click the debug icon ![Choose React Native debugger](images/debug-view-icon.png) in the View bar, and then click the configuration (gear) icon ![Configure-gear](images/configure-gear-icon.png), then choose the React Native debug environment.
```

Comment:
I dont think this works. I havent seen this in my VSCode install on my personal laptop.. Are we sure this works?

Reply:
@dhanvikapila Yes, it should work, otherwise it's an VS Code issue. Also, please, be aware that menu will not be shown unless you already have `launch.json` - VS Code opens this file instead.